requirements.txt

```
@@ -8,3 +8,4 @@ phx-class-registry~=4.0
 rst2ansi==0.1.5
 tabulate==0.9.0
 tqdm~=4.65.0
+platformdirs==4.0.0
```

```suggestion
platformdirs~=4.0.0
```
tight dependency constraints tend to be bad for anyone that wants to install this as python package. It also means we have to update it manually very often. Which is why we have started relaxing those as you can see above for new additions.